ide-common/src/main/kotlin/org/digma/intellij/plugin/codelens/AbstractCodeLensService.kt

```diff
@@ -147,6 +147,7 @@ abstract class AbstractCodeLensService(private val project: Project): Disposable
         // try to find a replacement for CodeVisionPassFactory.clearModificationStamp and refresh only one by one.
 
         if(project.isDisposed){
+            Log.test(logger::info, "Project already disposed")
             return
```

Comment:
this log message gives no information but that the project is disposed but not why its important to know about it